pkg/semtech-loramac/contrib/semtech_loramac.c

```diff
@@ -0,0 +1,624 @@
+/*
+ * Copyright (C) 2017 Inria
+ *               2017 Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     pkg_semtech-loramac
+ *
+ * @file
+ * @brief       Implementation of public API for Semtech LoRaMAC
+ *
+ *              This implementation is an adaption of the applications provided
```

Comment:
Seeing that you're not consistently following this indentation, please fix the long descriptions to align with the "@".

Reply:
I agree with @kaspar030, long/detail description must not align with `@brief`, but rather should align to one space after `*`, i.e., with the `@`s above and below for params.